src/base.ts

```diff
@@ -184,9 +184,23 @@ export abstract class Base<T extends IBaseSettings = IBaseSettings> {
     return this.render();
   }
 
-  remove() {
-    this.settings.map.removeLayer(this.layer as any);
-    this.active = false;
-    return this;
+  remove(indices: any) {
```

Comment:
I believe this should be `  remove(indices?: number) {`  and rather than check if it is undefined, check if it is a number, and perform the removal, otherwise, perform what is now the undefined check.  That way there is only one check.

Reply:
`indices` can also be an array of numbers, so the second check casts a number to an array, otherwise the `sort+forEach` wouldn't work.

So I could change to `remove(indices?: number|number[])`  or just `remove(indices?: number[])` and leave the function body unchanged.
Or would you like the function to only accept 1 number?

Reply:
`number | number[]` would be preferred, ty!